lib/JSON/Validator.pm

```diff
@@ -42,6 +42,8 @@ has cache_paths => sub {
 
 has formats => sub { shift->_build_formats };
 
+has recursive_data_protection => sub { return 1 };
```

Comment:
No need for sub, nor return. Just do `has recursive_data_protection => 1;`

Reply:
Changed to a straight value.